src/query_utils.cc

```diff
@@ -245,6 +245,28 @@ std::vector<QueryLocation> GetDeclarationsOfSymbolForGotoDefinition(
   return {};
 }
 
+std::vector<QueryLocation> GetGotoDefinitionTargets(
```

Comment:
I've been moving things out of query_utils.cc if they are only used in one file. Can you move this to text_document_definition.cc?

Reply:
Done